sensor/kubernetes/fake/deployment.go

```diff
@@ -35,7 +35,12 @@ func createRandMap(stringSize, entries int) map[string]string {
 }
 
 func (w *WorkloadManager) getDeployment(workload DeploymentWorkload) *deploymentResourcesToBeManaged {
-	labels := createRandMap(16, 3)
+	var labels map[string]string
+	if workload.NumLabels == 0 {
+		labels = createRandMap(16, 3)
```

Comment:
We already talked about this in the call but just to have record here.

The group of deployments that has 10 labels won't be randomize. We want this to make the labels match with the `NetworkPolicies`. I can think of a lot of improvements for this, for example adding a new field indicating if we want them to be random (e.g. `randomLabels`). We could also add another one to add a prefix to have more control on how they created and how they match. But since this is time sensitive for the release I'm not sure if we should address these things now.